cmd/snap/cmd_delete_key_test.go

```diff
@@ -44,6 +45,11 @@ func (s *SnapKeysSuite) TestDeleteKeyNonexistent(c *C) {
 }
 
 func (s *SnapKeysSuite) TestDeleteKey(c *C) {
+	// FIXME: remove once xenials gpg2 is fixed
```

Comment:
Yeah, its mostly a issue during the tests though, when this is used interactively it should not be a concern because the agent will prompt if its ok to remove the key which is one additional step but it will work.